lib/tasks/update_banned_url_shorteners.rake

```diff
@@ -0,0 +1,40 @@
+# typed: false
+
+require "open-uri"
+
+desc "Update banned list of URL shorteners"
+
+task update_banned_url_shorteners: :environment do
+  url = "https://raw.githubusercontent.com/mayakyler/link-shorteners/refs/heads/main/js-link-shorteners/src/link-shorteners.txt"
+
+  begin
+    response = URI.parse(url).open
+    content = response.read
+
+    domains = content.split("\n").reject { |line| line.empty? || line.starts_with?("#") }
```

Comment:
Why did you write this `reject` filter?

Reply:
Oops. I was checking Peter Dave's Link Shortener list. Had to used it there. Removing it.